runtime/shared_common/shrinit.cpp

```diff
@@ -3177,6 +3177,44 @@ j9shr_isBCIEnabled(J9JavaVM *vm)
 	return (0 != (vm->sharedClassConfig->runtimeFlags & J9SHR_RUNTIMEFLAG_ENABLE_BCI));
 }
 
+static UDATA
+romToRamGetRomAddress(void *item) {
+	RomToRamQueryEntry *queryEntry = (RomToRamQueryEntry *)item;
+	UDATA romAddress = (UDATA)(queryEntry->romClass);
+	if (J9_ARE_ALL_BITS_SET(romAddress, ROM_TO_RAM_QUERY_TAG)) {
+		return romAddress ^ ROM_TO_RAM_QUERY_TAG;
```

Comment:
Shouldn't this be `(romAddress & ~ROM_TO_RAM_QUERY_TAG)` ?

Reply:
They're equivalent at that point because of the guard, but `(romAddress & ~(UDATA)ROM_TO_RAM_QUERY_TAG)` is more consistent with the semantics and the rest of the code base.